theano/tensor/nnet/tests/test_abstract_conv.py

```diff
@@ -564,13 +597,21 @@ def test_all(self):
             for provide_shape in self.provide_shape:
                 yield (self.tcase, i, f, ds, db, dflip, provide_shape)
             if min(i) > 0 and min(f) > 0:
-                for fd in self.filters_dilations:
-                    for s in self.subsamples:
-                        for b in self.border_modes:
-                            yield (self.tcase, i, f, s, b, dflip,
-                                   dprovide_shape, fd)
-                for flip in self.filter_flip:
-                    yield (self.tcase, i, f, ds, db, flip, dprovide_shape)
+                for u in self.unshared:
+                    if u and len(i) != 4:
+                        continue
```

Comment:
Can you add a comment saying that this is because it's not yet implemented in 3D?